bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/TombstoneParser.kt

```diff
@@ -28,11 +30,42 @@ internal class TombstoneParser(
             if (listOpenFds) {
                 extractTombstoneFd(trace.openFdsList, fileDescriptorConsumer)
             }
+
+            if (includeLogcat) {
+                extractTombstoneLogBuffers(trace.logBuffersList, logcatConsumer)
+            }
         } catch (ex: Throwable) {
             logger.w("Tombstone input stream threw an Exception", ex)
         }
     }
 
+    private fun extractTombstoneLogBuffers(
+        logBuffersList: List<TombstoneProtos.LogBuffer>,
+        logcatConsumer: (String) -> Unit
+    ) {
+        val newLogList = mutableListOf<String>()
+        var priorityType: String
+        logBuffersList.forEach { logs ->
+            logs.logsList.forEach {
+                priorityType = when (it.priority) {
+                    2 -> "VERBOSE"
+                    3 -> "DEBUG"
+                    4 -> "INFO"
+                    5 -> "WARN"
+                    6 -> "ERROR"
+                    7 -> "ASSERT"
```

Comment:
Rather than literals these should be `const val`s with a comment that they are copied from `android.util.Log`